dma-side-channel/main.c

```diff
@@ -0,0 +1,78 @@
+#include <stdint.h>
+#include <msp430.h>
+#include <sancus/sm_support.h>
+#include <sancus_support/sm_io.h>
+#include <sancus_support/sancus_step.h>
+
+#define DELAY_BEFORE_SM 30
+#define INSTRUCTION_NUMBER_JMP 36
+
+int instruction_counter = 0;
+uint16_t **dma_addr     = (uint16_t**) 0x0070;
```

Comment:
probably makes sense to have these MMIO addresses as constants in a header file in sancus-support? eg have a look at the tsc peripheral example? If we decided to move the peripheral in the address space we'll only have to update one file ^^

Reply:
Pull request submitted for the changes in sancus-support: https://github.com/sancus-pma/sancus-support/pull/10